res2df/pvt.py

```diff
@@ -299,7 +306,14 @@ def pvt_main(args) -> None:
     resdatafiles = ResdataFiles(args.DATAFILE)
     logger.info("Parsed %s", args.DATAFILE)
     if resdatafiles:
-        deck = resdatafiles.get_deck()
+        try:
+            deck = resdatafiles.get_deck(
+                sections=[opm.io.eclSectionType.RUNSPEC, opm.io.eclSectionType.PROPS]
+            )
+        except (
```

Comment:
You may want to move this `except` to inside `get_deck()`?

Reply:
Will take a look at it, probably a good idea.

Reply:
seems like an issue in this case will be that we cannot run it with opm < 2024.04 (which doesn't exist yet), as   `opm.io.eclSectionType.RUNSPEC` isn't valid (without complicating the code outside get_deck). 
An option is to instead have a list of strings as the `get_deck()` input and then map them to `opm.io.eclSectionType` inside `get_deck()`. Might also make it easier for most users. 
E.g. a syntax like this `get_deck(sections=["RUNSPEC", "PROPS"]`)